### PR TITLE
Properly catch exception on validateToken

### PR DESCRIPTION
**Pull request by ramzimaalej**

### CODE REVIEW

File: generators/server/templates/src/main/java/package/security/jwt/_TokenProvider.java [hunk 1]
```diff
@@ -90,7 +90,15 @@ public boolean validateToken(String authToken) {
             return true;
         } catch (SignatureException e) {
             log.info("Invalid JWT signature: " + e.getMessage());
```
Comment (gmarziou) [inline generators/server/templates/src/main/java/package/security/jwt/_TokenProvider.java]: Why not logging full exception:?

    log.info("Invalid JWT signature", e);

Comment (ramzimaalej) [inline generators/server/templates/src/main/java/package/security/jwt/_TokenProvider.java]: I just kept the initial logging. But, definitely, I can log the exception as requested

**Comment by cbornet** [comment]
With tests ! Very nice !

**Comment by jdubois** [comment]
Yes it's better, but I don't think this solve the "500" error

**Comment by ramzimaalej** [comment]
The fix prevents the "500" error as the exception is caught and a false is returned thus denying access to the requested resource. I'll try to add an integration test to capture that.

**Comment by jdubois** [comment]
Thanks, I'm on my phone so it's probably just me having trouble to figure out how this works. I'll test it today.

**Pull request merged**